benchmarks/graph_infra/arkouda.graph

```diff
@@ -124,6 +124,12 @@ files: flatten.dat, flatten.dat, flatten.dat
 graphtitle: Flatten Performance
 ylabel: Performance (GiB/s)
 
+perfkeys: _get_head_tail_server Average rate = , _get_head_tail Average rate =
+graphkeys: _get_head_tail_server Average rate GiB/s, _get_head_tail GiB/s
```

Comment:
should maybe be `_get_head_tail Average rate GiB/s` to be consistent with other graphkey

Reply:
updated.